src/oop/user_defined_classes/medium/q2/QuestionText.md

```diff
@@ -0,0 +1,25 @@
+# oop-user_defined_classes-medium-q2
+
+## Java
+
+Little boys love to play with cars and trucks and watch things move. 
```

Comment:
I think for this question, we can have them also add the attributes, not just the method.